dolo/compiler/model_numeric.py

```diff
@@ -179,6 +179,14 @@ def _repr_html_(self):
             equations = {"dynare": self.symbolic.equations}
         else:
             equations = self.symbolic.equations
+            # Create definitions equations and append to equations dictionary
+            definitions = self.symbolic.definitions
+            tmp = []
+            for deftype in definitions:
+                tmp.append(deftype + '=' + definitions[deftype])
+            definitions = {'definitions': tmp}
+            equations.update(definitions)
```

Comment:
Ok cool. I can't remember exactly, but I think without setting the local `equations` to `copy(self.symbolic.equations)` or `deepcopy(self.symbolic.equations)` we will be modifying the original one, even outside the display function.

We should double check that before we merge this